### ISSUE TITLE
Version 12 has 'object' as type, but should be array of type.

### ISSUE THREAD

**Issue by frankhommers** [ccouzens/keycloak-openapi#9]
I think it's a wider problem than only this, I think if you solve it for this, it will be solved for all:

In v12 this 

Get identity providers
`GET /{realm}/identity-provider/instances`

becomes an object in de openapi json.

But it was in v11 still an array of IdentityProviderRepresentation as it should be.

v12:
https://github.com/ccouzens/keycloak-openapi/blob/b865c669d9c250b0ba1a26ada31616f4bec2326a/keycloak/12.0.json#L6963

v11:
https://github.com/ccouzens/keycloak-openapi/blob/b865c669d9c250b0ba1a26ada31616f4bec2326a/keycloak/11.0.json#L7103

**Comment by ccouzens** [comment]
Hi @frankhommers ,

Thanks for reporting this 🙂

I've changed it so that `Stream` in keycloak's documentation produces the following json:

```json
                 "schema": {
                  "type": "array",
                  "items": {
                    "type": "object",
                    "additionalProperties": true
                  }
                 }
```

I can't make it into an array of `IdentityProviderRepresentation` because the documentation doesn't have the information in it 🙁 

![Screenshot_2021-01-15 Keycloak Admin REST API](https://user-images.githubusercontent.com/241046/104769122-b16e2b00-5766-11eb-99e7-0f9c7b3e9fe7.png)

**Comment by ccouzens** [comment]
https://github.com/ccouzens/keycloak-openapi/compare/b865c669d9c250b0ba1a26ada31616f4bec2326a...5449ed5edb70f9693d9e120bed649decd576eaa1

**Comment by frankhommers** [comment]
Yeah I see it... In my opinion that's a bug in their documentation. This change to stream made it pretty useless :(

**Comment by ccouzens** [comment]
I've created a patched version of 12.0, so that these fixes can be made manually.

https://github.com/ccouzens/keycloak-openapi/blob/master/keycloak/12.0-patched.json

So far, it just fixes the return type of `/{realm}/identity-provider/instances` and `/{realm}/clients`.

I've also made a "help wanted" issue for someone other than me to continue making fixes such as this https://github.com/ccouzens/keycloak-openapi/issues/10

**Comment by frankhommers** [comment]
I did the same but for much more. I opened 12.json, 11.json and 10.json, and merged everything applicable into 12.json. I'll fork.
